src/xai_grok/errors/incomplete_request_error.py

```diff
@@ -0,0 +1,10 @@
+class IncompleteRequestError(Exception):
+    """
+    Error occurs when request body is missing fields
+
+    params:
+        message: the error message
+    """
+
```

Comment:
The docstring incorrectly uses 'params' instead of 'Attributes' or 'Args' to describe the constructor parameter, which may confuse developers about the class's usage.